README.md

```diff
@@ -1,89 +1,74 @@
 
-### OAuth Requirements
+Autolinking should install the CocoaPods dependencies for iOS project. If it fails you can run 
+
+```sh
+cd ios && bundle install && bundle exec pod install
+```
+
+#### OAuth Requirements
 
 :warning: All integrations must migrate to version 9.0.0 or later of the React Native SDK (requires version 4.1.0 or later of the iOS LinkKit SDK) by September 30, 2023, to maintain support for Chase OAuth on iOS. 
 
 For iOS OAuth to work, specific requirements must be met.
-* Redirect URIs must be registered, and set up as universal links ([docs](https://plaid.com/docs/link/ios/#register-your-redirect-uri))
+* Redirect URIs must be [registered](https://plaid.com/docs/link/ios/#register-your-redirect-uri), and set up as [universal links](https://developer.apple.com/documentation/xcode/supporting-associated-domains).
+
 
+### Android Setup
 
-## Android setup
-[Autolinking][autolinking]  unfortunately isn't supported for Android, because the SDK makes use of [TurboModules][turbomodules] to speed up app cold starts. Have a look at [#306](https://github.com/plaid/react-native-plaid-link-sdk/pull/306) for more context.
+- Android 5.0 (API level 21) and above.
+  - Your `compileSdkVersion` must be `33`.
+- Android gradle plugin `4.x` and above.
 
-### 1. Register your app id
-1. Log into your [Plaid Dashboard][plaid_dashboard] at the API page
+AutoLinking should handle all of the Android setup. 
+
+#### OAuth Requirements
+
+##### Register your app id
+1. Log into your [Plaid Dashboard](https://dashboard.plaid.com/team/api) at the API page
 2. Next to Allowed Android package names click "Configure" then "Add New Android Package Name"
 3. Enter your package name, for example `com.plaid.example`
 4. Click "Save Changes", you may be prompted to re-enter your password
 
-### 2. Add PlaidPackage to your application
-1. Go to `android/app/src/main/java/<AppName>/MainApplication.java`
-2. Add `import com.plaid.PlaidPackage;` to the imports section
-3. Add `packages.add(new PlaidPackage());` to `List<ReactPackage> getPackages();`
 
-### 3. Configure Gradle
-1. Go to the project level `android/app/build.gradle`
-2. Make sure you are using a min sdk >= 21
-3. Add the following dependencies:
+### React Native Setup
 
-```groovy
-dependencies {
-    // ...
-    implementation project(':react-native-plaid-link-sdk')
-}
-```
-
-5. Go to `android/settings.gradle`
-6. Add the following lines:
-
-```groovy
-include ':react-native-plaid-link-sdk'
-project(':react-native-plaid-link-sdk').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-plaid-link-sdk/android')
-```
-4. (Optional) Run `npx react-native link`
+- To initialize `PlaidLink`, you will need to first create a `link_token` at [/link/token/create][https://plaid.com/docs/#create-link-token]. Check out our [QuickStart guide](https://plaid.com/docs/quickstart/#introduction) for additional API information.
```

Comment:
oops, formatting for this link is broken (should be parens instead of square brackets)